### DISCUSSION TITLE
Option To Allow Videos To Play Entire Length

### DISCUSSION THREAD

**Discussion by ghost** [Geektoolkit/Dynaframe3#115] [Ideas]
I have playlists that include images and video clips interspersed between.
It would be nice to be able to have the ability to have duration set for images, but the videos play their entire length before continuing on to the next slide.

**Comment by dankinchen** [comment]
But then you would have to figure out how to time it with syncing? I guess there would have settings for multiple frames and separate settings for single frames.

**Comment by qwksilver** [comment]
sync timing is getting revamped soon, when it is there will be things like autopause on sync frames, and better timing sync for video sets, and finally there will be a backup timer that will rotate the photo from the local playlist if an incoming timing message is missed for too long.

**Reply by Geektoolkit** [reply]
qksilver's correct, syncing will need to be revamped. For now the option is there to play a full video, so this workitem is completed.
For syncing, it needs to be reworked from where it's at in 2.22 since Video playback was updated to be in it's own class. If you have the setting to fully play videos you'll likely go out of sync, and I wouldn't try to fix that. The idea behind syncing is that things are curated to match...either so that images can go multiframe (or videos once the sync timer is in) or so that the 'theme' of a room can match. For instance if I set a Marvel theme in a room, videos and iamges should be marvel, and if they get a bit out of sync but are still Marvel based then I consider that ok.  If you're trying to sync tighter (like multiscreen images) curation will be required unless someone has a better way to propose that can be coded up reliably :)
